src/internal/feature/feature.go

```diff
@@ -0,0 +1,213 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package feature provides feature flags.
+package feature
+
+import (
+	"fmt"
+	"sync/atomic"
+)
+
+// Atoms wrapping the default and user-set collections of Features. These do not require mutexes because they will each
+// be modified very few times, with the public API for each ensuring there only an empty set can be written to.
```

Comment:
nit: potentially some grammar fixes (Or I learn how to read). 

> ensuring there only an empty set can be written to

Reply:
Fixed! There -> that